ResearchKit/+Medable/ORKAutocompleteStep/ORKAutocompleteStep.m

```diff
@@ -10,6 +10,7 @@
 #import "ORKAnswerFormat.h"
 #import "ORKAutocompleteStepViewController.h"
 
+
```

Comment:
We should avoid committing non-changes like this to files.

I don't mean a stray line addition/deletion is bad, just saying that shouldn't be all that has changed in the file (one instance of each in this case)